tests/integration/conftest.py

```diff
@@ -30,6 +30,9 @@ def pytest_configure(config):
     os.environ['TARGETURL'] = "http://localhost:8000/"
     start_gunicorn()
 
```

Comment:
Likely to break in 1.9, for `bin_test.py` at the least, since every binary also bootstraps. I've been trying to add the bootstrap as closely as possible to where it is needed. It needs to go before first use of anything needing anything from `settings`, and before any import of a model. Sometimes the guard in the bootstrap-code seems to work, and sometimes it doesn't, so it's iffy when it is allowable/possible to call bootstrap twice.

Reply:
Not sure why this would fail. The binaries bootstrap, yes, but they are separate processes.

Reply:
Grr, this is not what I originally commented on. The point however is that I run `bootstrap_django` as close to where it is needed as possible, and bootstrapping twice sometimes breaks sometimes does not break, and Django 1.9 is even more picky.